code/LeftAndMain.php

```diff
@@ -1889,13 +1889,15 @@ public function getHelpLinks()
             }
         });
 
-        foreach ($helpLinks as $key => $value) {
-            $translationKey = str_replace(' ', '', $key ?? '');
-
-            $formattedLinks[] = [
-                'Title' => _t(__CLASS__ . '.' . $translationKey, $key),
-                'URL' => $value
-            ];
+        if (is_array($helpLinks)) {
```

Comment:
Add a comment here and in the PHPDoc for the `help_links` property that this can be set to `false` to disable it.

Reply:
Done